Sources/NIOHTTPCompression/HTTPDecompression.swift

```diff
@@ -86,9 +86,7 @@ public enum NIOHTTPDecompression {
         }
 
         mutating func decompress(part: inout ByteBuffer, buffer: inout ByteBuffer, originalLength: Int) throws {
-            buffer.reserveCapacity(part.readableBytes * 2)
-
-            self.inflated += try self.stream.inflatePart(input: &part, output: &buffer)
+            self.inflated += try self.stream.inflatePart(input: &part, output: &buffer, minimumCapacity: part.readableBytes * 2)
```

Comment:
`part` is passed `inout`, so we don't need the minimum capacity here: we can push it down into `inflatePart` .

Reply:
good idea, fixed